packages/sewing-kit-koa/src/middleware.ts

```diff
@@ -30,11 +31,20 @@ export function setAssets(ctx: Context, assets: Assets) {
 export default function middleware({
   serveAssets = false,
   assetPrefix = defaultAssetPrefix(serveAssets),
+  assetPrefixCallback,
   manifestPath,
 }: Options = {}): Middleware {
   async function sewingKitMiddleware(ctx: Context, next: () => Promise<any>) {
+    let currentAssetPrefix;
+
+    if (!serveAssets && assetPrefixCallback) {
+      currentAssetPrefix = assetPrefixCallback(ctx);
+    } else {
+      currentAssetPrefix = assetPrefx;
+    }
+
     const assets = new Assets({
-      assetPrefix,
+      currentAssetPrefix,
```

Comment:
I don't see the matching change to `Assets`'s constructor arguments? I kind of feel like I preferred the old name too /shrug

Reply:
Ah 😿 I had to change this because lint was complaining about reassigning the `assetPrefix` param. Oops just realised this is a mistake as well.